nifi-extension-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/AbstractExecuteSQL.java

```diff
@@ -531,4 +554,38 @@ protected List<String> getQueries(final String value) {
     }
 
     protected abstract SqlWriter configureSqlWriter(ProcessSession session, ProcessContext context, FlowFile fileToProcess);
+
+    enum ContentOutputStrategy implements DescribedValue {
+        EMPTY_RESULT(
+            "Overwrite Content",
+            "Overwrites the FlowFile content with the empty result set."
+        ),
+        IGNORED(
+            "Ignore Results",
+            "Ignores the result and passes the incoming FlowFile content to the next processor."
+        );
+
+        private final String value;
+        private final String description;
+
+        ContentOutputStrategy(final String value, final String description) {
+            this.value = value;
+            this.description = description;
+        }
+
+        @Override
+        public String getValue() {
+            return this.value;
```

Comment:
This should use the `name()` method to align with the enum.
```suggestion
            return name();
```